frontend/src/features/dsComponents/body/BodyPreview.tsx

```diff
@@ -1,15 +1,12 @@
 // TODO(chriswhong): BodyPreview and Body should be combined since they share a lot of the same logic
 
-import React, { useEffect } from 'react'
+import React from 'react'
```

Comment:
wait, so we have a `BodyPreview` component and a `Body` component with a `preview` prop? Do we need both?

Reply:
No, these should be consolidated.  I think it can happen in this issue: https://github.com/qri-io/qrimatic/issues/158